llvm/utils/git/github-automation.py

```diff
@@ -635,16 +635,16 @@ def execute_command(self) -> bool:
         """
         This function reads lines from STDIN and executes the first command
         that it finds.  The 2 supported commands are:
-        /cherry-pick commit0 <commit1> <commit2> <...>
```

Comment:
Do not include unrelated things into a PR

Reply:
Hello, I was assigned two issues and I have solved both of them to the best of my knowledge. And I pushed the patch for the second issue after solving the first one. Since I can't create a new PR when my first PR has not been merged yet, So I thought of just pushing it as I wanted to work on more issues. The issues are : [this](https://github.com/llvm/llvm-project/issues/58290#issuecomment-1886650144) and [this](https://github.com/llvm/llvm-project/issues/64803)

Reply:
Since @tbaederr was involved in both the issues, I hope both the patches can be considered in a single PR.

Reply:
You certainly could submit multiple PRs for separate changes.